cmd/mattermost/commands/utils.go

```diff
@@ -99,3 +100,19 @@ func printStringMap(value reflect.Value, tabVal int) string {
 
 	return out.String()
 }
+
+func getConfigDSN(command *cobra.Command, env map[string]string) string {
+	configDSN, _ := command.Flags().GetString("config")
+
+	// Config not supplied in flag, check env
+	if configDSN == "" {
+		configDSN = env["MM_CONFIG"]
```

Comment:
Will `MM_CONFIG` key will be existing all the time?

Reply:
No, but accessing a non existent field will return "" which is expected here.